packages/web-components/docs/coding-conventions.md

```diff
@@ -437,15 +439,38 @@ To cause re-rendering upon change in `private`/`protected` properties, use `stat
 
 Global `document`/`window` can be different from the ones associated with custom element instance, when the custom element is transported to a different frame e.g. with `document.importNode()`. Though such cases are rare, the codebase avoids global `document`/`window` reference to keep ourselves in a good DOM citizen. We use [`element.ownerDocument`](https://developer.mozilla.org/en-US/docs/Web/API/Node/ownerDocument)/[`.element.ownerDocument.defaultView`](https://developer.mozilla.org/en-US/docs/Web/API/Document/defaultView), respectively, instead.
 
-## Custom element registration
+## Custom Elements
+### @carbonElement Decorator
+
+When creating a custom element, we alias `@carbonElement` as lit framework's `@customElement`. Lit's `@customElement` does not provide
+a way to check if an element has already been defined. If an attempt is made to define an element that has already been defined on a page, an error state will occur and code will break. 
+
+This new approach offers a method for elements to attempt to define themselves, but to gracefully handle the failure and to continue executing the remainder of the running script. The new decorator is a simple copy/paste from the LitElement `@customElement` decorator, with the `customElements.define` functions wrapped in a try/catch block.
```

Comment:
The wording in the first sentence was a little misleading if you read it literally, and I think removing any notes about the implementation is a good idea since theoretically that could change. Here's what I came up with:

```suggestion
We use a custom `@carbonElement` decorator instead of Lit's `@customElement`, which doesn't provide
a way to check if an element has already been defined in the window's `CustomElementRegistry`. If an attempt is made to redefine an element that has already been registered, an error will occur and any remaining code will fail to execute.

`@carbonElement` is identical to `@customElement` with the exception that it gracefully handles any failures from duplicate registrations and continues executing the remainder of the running script.

#### Usage
```